teachopencadd/talktorials/T013_query_pubchem/talktorial.ipynb

```diff
@@ -6,10 +6,13 @@
    "source": [
```

Comment:
<blockquote>In this section, we will discuss how to search compounds from PubChem <s>with</s> <strong>based on a </strong>given <s>structure</s> <strong>molecule </strong><s>by</s> <strong>using the </strong> PUG-REST access.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>done</p>